README.md

```diff
@@ -11,7 +11,9 @@ uses [React](https://reactjs.org/).
 Download the official installer for your operating system:
 
  - [macOS](https://central.github.com/deployments/desktop/desktop/latest/darwin)
+ - [macOS (Apple Silicon)](https://central.github.com/deployments/desktop/desktop/latest/darwin-arm64)
  - [Windows](https://central.github.com/deployments/desktop/desktop/latest/win32)
+ - [Windows (ARM64)](https://central.github.com/deployments/desktop/desktop/latest/win32-arm64)
```

Comment:
```suggestion
```

As mentioned in Slack, the Windows ARM support is not finished nor official yet, since it still uses x64 git to work, but there is work in progress! 😄